docs/contracts/inheritance.rst

```diff
@@ -297,6 +297,43 @@ The reason for this is that ``C`` requests ``X`` to override ``A``
 requests to override ``X``, which is a contradiction that
 cannot be resolved.
 
+One area where inheritance linearization is especially important and perhaps not as clear is when inheriting and implementing multiple `constructor`s. The linearization will apply the `constructor`s in the order of the inheritance not in the order in which they are declared in the inheriting class's `constructor`.  For example:
```

Comment:
My gut feeling here would be to just use the plain english word "constructor", i.e. in regular font and not in code-type.

The first sentence could maybe be simplified my combining "inheriting and implementing" into "when there are multiple constructors in the inheritance hierarchy."

The second sentence could be simplified to "The constructors will always be executed in the linearized order, regardless of the order in which their arguments are provided in the inheriting contract's constructor."